lib/credit_card_validations/plugins/diners_us.rb

```diff
@@ -0,0 +1,5 @@
+CreditCardValidations.add_brand(
+    :diners_us,
+    { length: 16, prefixes: %w(54 55) },
+    { brand_name: 'Diners Club US' }
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.